kubernetes_cluster/main.tf

```
@@ -40,14 +40,14 @@ resource "azurerm_kubernetes_cluster" "this" {
   }
 
   role_based_access_control {
-    enabled = var.rbac_enabled
+    enabled = var.rbac_enabled #tfsec:ignore:AZU007
```

this should be enabled by default
```suggestion
    enabled = true
```

---

Agree fixed also misleading description

---

Agree fixed also misleading description